packages/eth-sdk/src/client/index.ts

```diff
@@ -19,7 +20,17 @@ export async function generateSdk(ctx: EthSdkCtx): Promise<void> {
   const outputToAbiRelativePath = relative(outputPath, abisRoot).replace(/\\/g, '/')
 
   const randomTmpDir = await fs.tmpDir('eth-sdk')
-  await generateTsClient(contracts, abisRoot, randomTmpDir, outputToAbiRelativePath, fs)
+
+  const shapedFlag: CodegenConfig = {
```

Comment:
I don't like that we provide defaults for the second time here. Could you make a PR to TypeChain that would export `DEFAULT_FLAGS`?  Then we could simply use them here and add only user provided flags